client/src/components/Overview/ImageGalleryThumbnailColumn.jsx

```diff
@@ -27,18 +27,22 @@ class ImageGalleryThumbnailColumn extends React.Component {
   }
 
   render() {
+    let index = 0;
     let thumbnails = this.props.productStyles.results[0].photos.map((style) => {
-      return <ImageGalleryThumbnail key={style.thumbnail_url} thumbnail_url={style.thumbnail_url} />
+      index += 1;
+      return <ImageGalleryThumbnail key={style.thumbnail_url} thumbnail_url={style.thumbnail_url} index={index - 1} gallery={this.props.gallery} />
```

Comment:
if you put the `index += 1` below the next line, you wouldn't have to decrement in the <ImageGalleryThumbnail> index prop.

Reply:
Can't put the increment after the return; that's why I did it that way. 
Of course there's always other ways, but they seemed equally convoluted in the end.